documentation/design-docs/dotnet-tools.md

```diff
@@ -396,6 +396,81 @@ CONVERT
       Writing:       ./trace.speedscope.json
       Conversion complete
 
+
+### dotnet-stack
+
+SYNOPSIS
+
+    dotnet-stack [options] [command] [<args>]
+
+OPTIONS
+
+    --version
+        Display the version of the dotnet-trace utility.
+
+    -h, --help
+        Show command line help
+
+COMMANDS
+
+    report         Displays stack traces for the target process
```

Comment:
nit: We've used the verb `collect` in our other tools. Perhaps switch for consistency's sake?

Reply:
My sense of past usage has been this:

- collect - capture data data write it to a file for offline analysis
- monitor - capture data and stream it to the console interactively
- report - capture data and display it on the console non-interactively

We only used 'report' in one other place (dotnet gcdump) so the trend isn't as well established. It seems like it fits to me, but I might also be splitting hairs that casual users aren't going to think about?

Reply:
Noah's logic is what led me to use `report`.  At some point, I'm hoping we might have a `report` verb on `dotnet-{trace|counters}` as well, that will allow you to produce views of data _a la_ `perf`'s `perf script ...` syntax.